website/docs/artist_hosts_nuke_tut.md

```diff
@@ -0,0 +1,166 @@
+In the Instance Creator, pick Create Write Render, and Create.
+
+![OpenPype Create](assets/nuke_tut/nuke_Create.png)
+![OpenPype Create](assets/nuke_tut/nuke_Creator.png)
+
+This will create a Group with a Write node inside.
+
+:::tip Admin Tip - Configuring write node
+You can configure write node parameters in **Studio Settings → Project → Anatomy → Color Management and Output Formats → Nuke → Nodes**
+:::
+
+## What Nuke Publish Does
+From Artist perspective, Nuke publish gathers all the stuff found in the Nuke script with Publish checkbox set to on, exports stuff and raises the Nuke script (workfile) version.
+The Pyblish dialog shows the progress of the process.
+
+![OpenPype Publish](assets/nuke_tut/nuke_Publish.png)
+
+![OpenPype Publish](assets/nuke_tut/nuke_PyblishDialogNuke.png)
+
+#### Publish steps
+- gathers all the stuff found in the Nuke script with Publish checkbox set to on (1)
+- collects all the info (from the script, database…) (2)
+- validates components to be published (checks render range and resolution...) (3)
+- extracts data from the script (generates thumbnail, creates reviews like h264 ...) (4)
+- copies and renames components (render, reviews, thumbnail, Nuke script...) to publish folder
+- integrates components (writes to database, sends preview of the render to Ftrack ...) (5)
+- increments Nuke script version, cleans up the render directory (6)
+
+#### Pyblish Note and Intent
+Artist can add Note and Intent before firing the publish (7) button. The Note and Intent is ment for easy communication between artist and supervisor. After publish, Note and Intent can be seen in Ftrack notes.
+
+#### Pyblish Checkbox
+Some may say Pyblish Dialog looks unnecessarily complex; it just tries to put a lot of info in a small area. One of the more tricky parts is that it uses non-standard checkboxes. Some squares can be turned on and off by the artist, some are mandatory.
+
+#### Pyblish Dialog
+The left column of the dialog shows what will be published. If you run the publish and decide to not publish the Nuke script, you can turn it off right in the Pyblish dialog by clicking on the checkbox. If you decide to render and  publish the shot in lower resolution to speed up the turnaround, you have to turn off the Write Resolution validator. If you want to use an older version of the asset (older version of the plate...), you have to turn off the Validate containers, and so on.
+
+Time wise, gathering all the info and validating usually takes just a few seconds, but creating reviews for long, high resolution shots can take too much time on the artist machine.
+
+More info about [Using Pyblish](artist_tools#using-pyblish)
+
+:::tip Admin Tip - Configuring validators
+You can configure Nuke validators like Output Resolution in **Studio Settings → Project → Nuke → Publish plugins**
+:::
+
+## Review
+When you turn the review checkbox on in your OpenPype write node, here is what happens:
```

Comment:
this could be perfectly supported by some Miro Board graphic to demonstrate the flow. like 
![image](https://user-images.githubusercontent.com/40640033/130219012-ccc8abc4-39ff-4b35-a2b4-0f4c91a54eb9.png)

Reply:
done